frontend/src/components/activePage/ActiveLanding/ActiveLanding.stories.tsx

```diff
@@ -0,0 +1,53 @@
+import ActiveLanding from './ActiveLanding';
+import { Story } from '@storybook/react';
+import { Title, Description, Stories } from '@storybook/addon-docs/blocks';
+
+export default {
+  title: 'Components/ActivePage/ActiveLanding/ActiveLanding',
+  component: ActiveLanding,
+  parameters: {
+    docs: {
+      page: () => {
+        <>
+          <Title />
+          <Description />
+        </>;
+      },
+    },
+  },
+  decorators: [
+    (Story: any) => (
+      <div style={{ margin: '2rem', height: '500px' }}>
```

Comment:
Why do you use this decorator? Also, on mobile/tablet it doesn't fit well

Reply:
Just to center the view in the Docs a bit and have a high-enough window to show the entire table without scrolling. I can remove the margin as it actually looks ok without.

Reply:
I would remove the margins